python/paddle/fluid/tests/custom_op/CMakeLists.txt

```diff
@@ -21,6 +21,12 @@ if(WITH_GPU OR APPLE)
   endif()
 endif()
 
+if(WITH_XPU)
+  set(CUSTOM_XPU_ENVS FLAGS_init_allocated_mem=0)
```

Comment:
init这个问题有找到具体原因吗？

Reply:
`py_test` 是 paddle 自己封装的函数（ref. `cmake/generic.cmake`）而非 cmake 内部函数，使用 `py_test` 添加单测时，默认开启 `FLAGS_init_allocated_mem=true` ，所以此处需要手动设置成 `FLAGS_init_allocated_mem=0` 避免报错

`py_test` is a function implemented in paddle (ref. `cmake/generic.cmake`) instead of cmake internal function. `py_test` uses `FLAGS_init_allocated_mem=true` by default, so we need to manually set `FLAGS_init_allocated_mem=0` in XPU unit test to avoid error.